util/query.py

```diff
@@ -458,6 +476,10 @@ def _queryNftvolumes(
                 continue
             basetoken_addr = order["lastPriceToken"]["id"].lower()
             nft_addr = order["datatoken"]["nft"]["id"].lower()
+            creator_addr = order["datatoken"]["nft"]["id"].lower()
```

Comment:
```suggestion
            creator_addr = order["datatoken"]["owner"]["id"].lower()
```

Reply:
I agree with this.

If we can move forward with this, then :+1: